CEXT-3925: Get all subscribed events & webhook endpoint (#306)
AdobeDocs/commerce-extensibility: src/pages/webhooks/api.md

---
title: REST Endpoints for Webhooks
description: Provides details about the webhook API endpoints.
keywords:
  - Webhooks
  - Extensibility
---

# REST endpoints for webhook.

Adobe Commerce provides several REST endpoints that interact with the webhooks processes. These endpoints require an [admin token](https://developer.adobe.com/commerce/webapi/rest/tutorials/prerequisite-tasks/).

## Get a list of all subscribed webhooks

> The `GET /rest/all/V1/webhooks/list` endpoint returns a list of all subscribed webhooks. The response body looks like below:

Reviewer: ```suggestion
The `GET /rest/all/V1/webhooks/list` endpoint returns a list of all subscribed webhooks. The response body is similar to the following:
```
